### PR TITLE
#1016 Update documentation

### PR DESCRIPTION
**Pull request by adriguir**
Update the documentation:

- Introduction modifications

- Models modifications

- Examples section to be introduced

- Section on the different tools

Will be completed by DynaWave topo missing, and other small modifications following final review. But you are already invited to give your advice.

@gautierbureau I would like to merge all the examples in one chapter for the global doc. Do you have a solution for this?

As it is it will create one chapter by example directory.

### CODE REVIEW

**Comment by adriguir** [comment]
The only remaining modification on my side would be to include the major changes from v1.2.0 that we need to decide upon.

File: documentation/dynawo_documentation.sh [hunk 1]
```diff
@@ -27,7 +28,8 @@ echo '%% Except where otherwise noted, content in this documentation is Copyrigh
 
 \begin{document}
 
-\title{\Dynawo Documentation \\ \small{v1.2.0}}
+\title{\huge{\Dynawo Documentation} \\ \LARGE{v1.2.0} \\
```
Comment (rosiereflo) [inline documentation/dynawo_documentation.sh:31]: Not related to this commit but it makes me think about it: I am going to modify the master version also (1.3.0 probably) to differentiate it from the release branches

File: documentation/functionalDoc/functionalDoc.tex [hunk 2]
```diff
@@ -40,7 +40,7 @@ \chapter{Functional documentation}
 \caption{Building a global model from different unit models in \Dynawo}
 \end{figure}
 
-To ensure acceptable performances both for compilation and simulation, a mechanism has been created into \Dynawo to compile non squared Modelica models individually (for example a generator model by itself) and before the simulation. These compiled models are then only instantiated during the simulation. This strategy enables to use \Dynawo for large-scale simulations (French EHV-HV networks - i.e. 250 000 continuous and 250 000 discrete variables) while keeping computation time in a range close to current simulation tools.\\
+To ensure acceptable performances both for compilation and simulation, a mechanism has been created into \Dynawo to compile non squared Modelica models individually (for example a generator model by itself) and before the simulation. These compiled models are then only instantiated during the simulation. This strategy enables to use \Dynawo for large-scale simulations (French EHV-HV networks, in the range of 100 continuous variables and 150 000 discrete variables after simplifications) while keeping computation time in a range close to current simulation tools.\\
```
Comment (rosiereflo) [inline documentation/functionalDoc/functionalDoc.tex:43]: I think there are more than 100 continuous variables in the french network :)

File: documentation/functionalDoc/functionalDoc.tex [hunk 3]
```diff
@@ -57,10 +57,130 @@ \chapter{Functional documentation}
...
+\includegraphics[width=\textwidth]{../resources/DynawoInitiative.png}
+\caption{High-level vision of the \Dynawo initiative}
+\end{figure}
 
 More details on the philosophy of \Dynawo could be found into \cite{Dynawo}.
 
+\newpage
+\subsection{DynaFlow}
+
+O\textbf{ne important and fundamental simulation tool for real-time operation and planning studies of the power system is the steady-state calculation tool.} This tool focuses on the correct calculation of the system equilibrium point following one or several event(s): it ensures the existence of a steady-state point and enables to check that this point is acceptable for the system operator. \\
```
Comment (rosiereflo) [inline documentation/functionalDoc/functionalDoc.tex:86]: O is outside the textbf

File: documentation/functionalDoc/functionalDoc.tex [hunk 4]
```diff
@@ -57,10 +57,130 @@ \chapter{Functional documentation}
...
+\end{center}
+
+\newpage
+\subsection{DynaSwing}
+
+\textbf{In order to protect the system against loss of synchronism or inter-area oscillations, the system operator has to conduct a large number of short-term stability studies. }Transient stability focuses on phenomena in the range of the ms and depends mainly on the active component models - synchronous machines and their controllers, HVDCs, Static Var Compensators. \\
+
+To properly represent the system response in the few instants following an event, \textbf{it is necessary to combine open models with detailed physical modelling and transparent block control scheme with a very accurate solving method.} It is the association of these different features that enable to catch the correct system fast dynamics. \\
+
+D\textbf{ynaSwing is a simulation tool that offers an unique combination of a transparent, flexible and robust way of modelling - thanks to the intensive use of the Modelica language - and a state-of-the-art, efficient and accurate numerical method (the variable time-step solver IDA).} \\
```
Comment (rosiereflo) [inline documentation/functionalDoc/functionalDoc.tex:154]: D outside the textbf

File: documentation/functionalDoc/functionalDoc.tex [hunk 5]
```diff
@@ -57,10 +57,130 @@ \chapter{Functional documentation}
...
+
+The development and release of some grid-forming controllers strategies and of dynamic lines are the first steps on which DynaWave will be built. They already enable to simulate in a completely open-source environment the three nodes test case used in the Migrate project to study interactions between different grid-forming controllers (the test case is under final validation but will be released soon). \textbf{The idea is to build upon this first basis to develop more open-source models, in collaboration with our partner and colleagues conducting researches on stability of power systems with a high-share of power-electronics based components.} DynaWave will provide a flexible and open-source simulation tool to ease technical discussions and collaborations in order to face the future stability challenges arising in power-electronics based system.
```
Comment (rosiereflo) [inline documentation/functionalDoc/functionalDoc.tex:177]: "They already enable " "are" is  missing

File: documentation/resources/dynawoDocumentation.bib [hunk 6]
```diff
@@ -148,3 +155,5 @@ @article{Hogan:2014:FRA:2639949.2560359
  address = {New York, NY, USA},
  keywords = {Adjoint code, Jacobian matrix, quasi-Newton, template metaprogramming},
 }
+
+@
```
Comment (rosiereflo) [inline documentation/resources/dynawoDocumentation.bib:159]: Useless @

**Comment by adriguir** [comment]
> Looks good! I detected a few typos

Thank you for having gone through it. I was expected a few ones, so thank you for having spotted them out. I will correct them asap and include the section on the main changes once we have agreed on them.

**Comment by adriguir** [comment]
Everything is finished for the documentation.
Following a discussion this morning with @gautierbureau it appears complicated to include at the moment the examples doc into the global documentation due to the relative paths existing.

What we propose is to distribute into the release the general documentation and the three pdf documentation related to the examples of DynaFlow, DynaWaltz and DynaSwing. We can perhaps zip the 4 pdfs in one zip in the distribution.

**Comment by rosiereflo** [comment]
ok let's do this for this release, we will be able to improve this later

**Comment by adriguir** [comment]
It's also what we agreed on.
If we find a solution soon, we will replace the documentation assets.

**Pull request merged**